CircleCI Windows build uses go1.15 instead of go1.16

**Describe the bug**
CircleCI uses go1.15 for it's Windows build instead of using go1.16.

**Steps to reproduce**
Put any code specific to go1.16 (e.g. using one of the functions moved out from the ioutil package, such as io.ReadAll) in a PR; this code will fail to build for this step.

**What did you expect to see?**
Code builds for Windows on CircleCI

**What did you see instead?**
Code fails to build for Windows on CircleCI:
```
.\client.go:103:17: undefined: io.ReadAll
.\client.go:115:17: undefined: io.ReadAll
note: module requires Go 1.16
```

**What version did you use?**
Occurs in PR #3884 

**What config did you use?**
N/A

**Environment**
Occurs in CI environment

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -195,7 +195,7 @@ commands:
       - run:
           name: Upgrade golang
           command: |
-            choco upgrade golang --version=1.15
+            choco upgrade golang --version=1.16
             refreshenv
       - run:
           name: Unit tests

